src/components/20-molecules/dropdown/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 7.0.1
```

Comment:
IMHO we don't need a changelog for internal refactorings. We also did not write changelog for all the releases we did, for instance, when someone is doing a global release.

Reply:
In this case there were actually changes in the dropdown component